modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSetDropContext.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import React from 'react';
+
+import {IFileDropSettings} from './utils/types';
+
+export type THandleFileDrop = (droppedItem: any, dropTarget?: any) => void;
+
+export interface IFrontendDataSetDropContext {
+	fileDropSettings: IFileDropSettings;
+	handleFileDrop: THandleFileDrop;
```

Comment:
For consistence, this should be `onFileDrop`. Do we need a dedicated function type? Typically an inline type is more then enough, something like
```js
interface IFrontendDataSetDropContext {
   onFileDrop: (droppedItem: any, dropTarget?: any) => void;
}
```